model/github/github_repository.rb

```diff
@@ -89,6 +79,16 @@ def setup_blob_storage
       update(access_key: token_id, secret_key: Digest::SHA256.hexdigest(token))
     end
   end
+
+  private def s3_client(access_key_id, secret_access_key)
+    Aws::S3::Client.new(
+      endpoint: Config.github_cache_blob_storage_endpoint,
+      access_key_id:, secret_access_key:,
```

Comment:
```suggestion
      access_key_id:,
      secret_access_key:,
      endpoint: Config.github_cache_blob_storage_endpoint,
```

Up to you, but I think this reads nicer.